src/AltinnCore/Common/Services/Implementation/AuthorizationStudioSI.cs

```diff
@@ -17,6 +17,7 @@ public class AuthorizationStudioSI : IAuthorization
         private const string TESDATA_USER_DIRECTORY = @"/User/";
 
         private const string REPORTEELIST_FILENAME = "reporteelist.json";
```

Comment:
I guess this line can be removed as there are no longer references to this file.